### PR TITLE
rm: habv4 secure boot: point to the secure machines page

### PR DESCRIPTION
**Pull request by vanmaegima**
We want the customers to walk through fusing and closing their board through the scripts we provide for each -sec machine, so point them to the other page.

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 7111c760ddf9a2615b406d83aea3b902b88253a9 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1798/docs/artifacts/html/index.html

**Comment by vanmaegima** [comment]
There's a lot of repeated information about checking our Secure Machines page but I don't know how to workaround this page to make it clear that this is only theoretical information for understanding and that we don't want customer running manual commands like shown in the page. I'd be happy to hear other opinions about this.

File: source/reference-manual/security/secure-boot-imx-habv4.rst [hunk 1]
```diff
@@ -292,17 +300,20 @@ To comply with these requirements we need to sign the image adding the ``--fix-s
 	$ ls SPL.signed
 	SPL.signed
 
-2.  **On newer SoCs** (ie imx7ulp), using the ``--fix-sdp-dcd`` parameter is not required.
+2.  **On newer SoCs**, like i.MX 7/8M and other i.MX 6 families, using the ``--fix-sdp-dcd`` parameter is not required.
```
Comment (angolini) [inline source/reference-manual/security/secure-boot-imx-habv4.rst:303]: I would drop the "older SoCs" and "newer SoCs" and keep only the family. As time here is not exactly right for all

**Review by angolini — APPROVED** [review]
LGTM, minor comment only

**Review by kprosise — COMMENTED** [review]
suggestions made, I will give it another read after suggestions are resolved to see if I missed anything. A number of the suggestions are not specific to this commit perse, and instead deal with the sentence the change is in.

**Comment by doanac** [comment]
Docs for 2ed850e4cc3b143b394b8b448e118fadb0ea5187 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1805/docs/artifacts/html/index.html

File: source/reference-manual/security/secure-boot-imx-habv4.rst [hunk 2]
```diff
@@ -184,16 +187,20 @@ Alternatively, use the kernel to program the A-core fuses using SDP via NXP's Un
...
 To secure the platform, there is an extra fuse that needs to be programmed: we will only take that step once we are sure that we can successfully sign and boot a signed SPL image with a matching set of keys (containing the same public key hashes as those stored in the SRK fuses).
 
-How to sign an SPL image (I)
-----------------------------
-To build a signed image, you need to create a Command Sequence File - CSF - describing all the commands that the ROM will execute during secure boot. These commands instruct HAB on which memory areas of the image to authenticate, which keys to install and use, what data to write to a register and so on. In addition, the necessary certificates and signatures involved in the verification of the image are attached to the CSF generated binary output.
+How to sign an SPL image
+------------------------
+
+.. note::
+    For simplicity, we provide a ``readme.md`` file with straight forward instructions on how to sign the SPL and mfgtool/SDP SPL for each board in our :ref:`ref-secure-machines` implementations. This is part of the ``mfgtool-files`` artifact for the secure machines.
```
[the remark this answers is not in the document]
Comment (vanmaegima) [reply source/reference-manual/security/secure-boot-imx-habv4.rst:201]: Dropped as the file in meta-lmp is called readme.md

**Comment by vanmaegima** [comment]
@kprosise applied suggestions from code review in a separate patch to simplify reviewing.

**Comment by kprosise** [comment]
@vanmaegima, LGTM on my end, and can be merged when you're ready.

**Pull request merged**